src/main/java/frc/robot/constants/ShooterConstants.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.constants;
+
+public class ShooterConstants {
+
+    // motor Ids (99 is placeholder for now)
+    public static final int BOTTOM_MOTOR_ID = 99;
+    public static final int TOP_MOTOR_ID = 99;
+
```

Comment:
-1 is usually a better placeholder, and put the PID constants and tolerance back in this constants file.

Reply:
Do not put PID constants here. They are local to the subsystem.